src/modules/attributes.ts

```diff
@@ -28,18 +32,18 @@ function updateAttrs(oldVnode: VNode, vnode: VNode): void {
         elm.setAttribute(key, "");
       } else if (cur === false) {
         elm.removeAttribute(key);
+      } else if (key.charCodeAt(0) !== xChar) {
+        elm.setAttribute(key, cur as any);
+      } else if (key.charCodeAt(3) === colonChar) {
+        key.charCodeAt(0) === sChar
+          ? elm.setAttributeNS(svgNS, key, cur as any)
+          : elm.setAttributeNS(xmlNS, key, cur as any);
+      } else if (key.charCodeAt(5) === colonChar) {
+        key.charCodeAt(1) === mChar
+          ? elm.setAttributeNS(xmlnsNS, key, cur as any)
+          : elm.setAttributeNS(xlinkNS, key, cur as any);
```

Comment:
The `// Assume xlink namespace` style comments in the original code are lost and they helped quite a bit to understand what's going on as the code itself takes some shortcuts.

Reply:
thanks for giving this PR your attention. I am glad to read your suggestions and will make a small update shortly when I have time